src/Input/Select/Select.stories.tsx

```diff
@@ -0,0 +1,210 @@
+import React from 'react';
+import { Story, Meta } from '@storybook/react';
+import Select, { Props as SelectProps } from './Select';
+import { WarningOutlineIcon } from '../../General/Icon/components';
+
+export default {
+  title: 'Input/Select',
+  component: Select,
+  argTypes: {
+    status: {
+      control: null,
+    },
+    error: {
+      control: { type: 'boolean' },
+    },
+  },
```

Comment:
You can wrap this into the `BaseContainer` to get the Poppins font. Looks better.
```suggestion
  decorators: [Story => <BaseContainer>{Story()}</BaseContainer>],
  },
```